docs/whirlpool/docs/05-More Resources/01-A Note for Rust Devs.mdx

```diff
@@ -0,0 +1,21 @@
+import DocCard from '@theme/DocCard';
+
+# A Note for Rust Developers
```

Comment:
Is this page still necessary since we have rust SDKs now? Guess this is fine for now but can you create a ticket for this?